src/redux/stores/config/configSlice.ts

```diff
@@ -18,6 +17,8 @@ const initialState: ConfigState = {
   ipfcGatewayURL: process.env.REACT_APP_IPFC_GATEWAY_URL || 'https://ipfs.io/ipfs/',
 };
 
+console.log(initialState);
```

Comment:
Is this console.log left on purpose or should be deleted?

Reply:
console.log should never be left!